prow/config.yaml

```diff
@@ -2656,6 +2656,40 @@ presubmits:
         resources:
           requests:
             memory: "6Gi"
+  - name: pull-kubernetes-e2e-kops-aws-coredns
+    agent: kubernetes
+    branches:
+    - release-1.10  # per-release image
+    max_concurrency: 12
+    always_run: true
```

Comment:
this probably either needs to be false or it needs to just be tested under the normal AWS job, IIRC the AWS accounts do not have enough quota for more than this, hence the `max_concurrency: 12` to throttle how many presubmits we run on AWS.